<?xml version="1.0"?>
<launch>

    <arg name="name" default="front" doc="Unique name to add to launched nodes (front or back)"/>

    <node name="synesthesiax_node" pkg="synesthesiax" type="synesthesiax_node" output="screen">
        <param name="max_range"        value="30" />
        <param name="min_range"        value="0.5" />
        <param name="max_ang_fov"   value=" 60" />
        <param name="min_ang_fov"   value="-60" />
        <param name="cloud_topic"   value="/ona2/sensors/pandar_$(arg name)/cloud" />
        <param name="img_topic"     value="/semantic_inference_$(arg name)/semantic/image_raw" />
        
        <rosparam command="load" file="$(find synesthesiax)/config/pinhole_model.yaml" />
    </node>

</launch>
